src/main/java/org/openhab/binding/zwave/handler/ZWaveSerialHandler.java

```diff
@@ -80,49 +89,86 @@ public void initialize() {
 
         portId = (String) getConfig().get(CONFIGURATION_PORT);
 
-        if (portId == null || portId.length() == 0) {
+        if (portId.isEmpty()) {
```

Comment:
I suggest that we make the port a required configuration in the thing XML file and then we can remove this check.

Reply:
I have checked the controller xml file and the port parameter was already defined as "required", so yeah, I suppose that at that point the portId cannot be empty, although I would prefer if we could leave it as it is, just in case something goes wrong.

Reply:
I don't see the need - it can never be null and adding the extra checks just makes the code less readable.